scripts/devnet

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+# Create a virtual bridge with PXE services and bootcfg
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+EXAMPLE=${2:-"etcd"}
+BRIDGE=metal0
+COREOS_VERSION=1153.0.0
+
+if [ "$EUID" -ne 0 ]
+  then echo "Please run as root"
+  exit
+fi
+
+function main {
+  case "$1" in
+    "create") create;;
+    "destroy") destroy;;
+    *)
+      usage
+      exit 2
+      ;;
+  esac
+}
+
+function usage {
+  echo "USAGE: ${0##*/} <command>"
+  echo "Commands:"
+  echo -e "\tcreate\tcreate a bridge with PXE services and CoreOS bootcfg"
+  echo -e "\tdestroy\tdestroy the bridge environment"
+}
+
+function check {
+  # SELinux, if present, it cannot be in Enforcing mode
+  if [ $(getenforce) == 'Enforcing' ]; then
+    echo "SELinux must be in permissive mode: 'setenforce Permissive'"
+    exit 1
+  fi
+
+  if [ ! -d $DIR/../examples/assets/coreos/$COREOS_VERSION ]; then
+    echo "Most examples use CoreOS $COREOS_VERSION. You may wish to download it."
+    exit 1
+  fi
+}
+
+function create {
+  check
+
+  echo "Starting bootcfg with examples/groups/$EXAMPLE mounted"
+  systemd-run --unit=dev-bootcfg \
+    rkt run \
+    --uuid-file-save=/tmp/bootcfg \
+    --net=metal0:IP=172.15.0.2 \
+    --mount volume=data,target=/var/lib/bootcfg \
+    --volume data,kind=host,source=$PWD/examples \
+    --mount volume=groups,target=/var/lib/bootcfg/groups \
+    --volume groups,kind=host,source=$DIR/../examples/groups/$EXAMPLE \
+    quay.io/coreos/bootcfg:latest -- -address=0.0.0.0:8080 -log-level=debug
+
+  echo "Starting dnsmasq to provide DHCP/TFTP/DNS services"
+  systemd-run --unit=dev-dnsmasq \
+    rkt run \
+    --uuid-file-save=/tmp/dnsmasq \
+    --net=metal0:IP=172.15.0.3 \
+    --mount volume=config,target=/etc/dnsmasq.conf \
+    --volume config,kind=host,source=$DIR/../contrib/dnsmasq/metal0.conf \
+    coreos.com/dnsmasq:v0.3.0
+}
+
+function destroy {
+  rkt stop --uuid-file=/tmp/bootcfg --force
```

Comment:
Without, if the stop hasn't finished it won't remove it. We just wanna remove it, we're only stopping it to be nice.

```
"2516b402-db39-45ea-bb95-dab8fe196822"
rm: pod "2516b402-db39-45ea-bb95-dab8fe196822" is currently running
rm: failed to remove one or more pods
```